src/App.vue

```diff
@@ -204,43 +394,134 @@ export default {
         // },
 
         favorites(favorites) {
-            localStorage.setItem('favorites', JSON.stringify(favorites))
+            localStorage.setItem('favorites', JSON.stringify(favorites));
 
-            return favorites
+            return favorites;
         },
     },
 
     methods: {
+        addColumn () {
+            this.inputColumn.push({id:(this.index++), name:'', type: ''});
+        },
+
+        removeRow (row) {
+            this.inputColumn.splice(row, 1);
+        },
+
+        toggleModal (title, add, type) {
+            this.showDialogCreate = !this.showDialogCreate;
+            this.dialogTitle = title;
+            this.showAddInput = add;
+            this.dataType = type;
+        },
+
+        closeModal () {
+            this.showDialogCreate = false;
+            this.newName.name = null;
+        },
+
+        capitalize (word) {
+            return word.charAt(0).toUpperCase() + word.slice(1);
```

Comment:
On cases like this, we normally use `lodash` to help us: https://lodash.com/docs/#capitalize